spec/forms/confirm_publish_course_form_spec.rb

```diff
@@ -50,6 +49,28 @@
           .from(nil).to((course.start_date + course.duration_in_years.years).to_date.prev_day)
       end
 
+      context "with itt_start_date set" do
+        let(:itt_start_date) { Time.zone.today }
+
+        it "updates all the course related attributes and itt_start_date" do
```

Comment:
Since the previous test already covers the other fields, you could trim this down to just the attribute of concern:

```suggestion
               it "updates all the course related attributes and itt_start_date" do
          expect { subject.save }
            .and change { trainee.course_start_date }
            .from(nil).to(itt_start_date)
        end
```

Reply:
fixed